CONTRIBUTING.md

```diff
@@ -1,5 +1,7 @@
 # Contributing guidelines
 
+Want to hack on kubernetes? Yay! We have a [Developer's Guide](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/README.md) that outlines everything you need to know from setting up your dev environment to how to get faster Pull Request reviews. And of course if you find something undocumented or incorrect along the way, feel free to send a PR!
```

Comment:
nit: relative link (`docs/devel/README.md`).

It also might fit better to add an h2 for this too. "Get hacking" or "Developer guide" or such maybe?